spec/system/search_spec.rb

```diff
@@ -85,4 +85,26 @@
       expect(search_page).to have_warning_message
     end
   end
+
+  describe "when search menu on desktop" do
+    before do
+      SearchIndexer.enable
+      SearchIndexer.index(topic, force: true)
+      SearchIndexer.index(topic2, force: true)
+    end
+
+    after { SearchIndexer.disable }
+
+    it "soft loads last topic results after navigating away, then back" do
```

Comment:
This description is abit unclear to me because I'm not quite sure what soft loads me.